src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Data/QueryFeaturesWithArcadeExpression/QueryFeaturesWithArcadeExpression.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.QueryFeaturesWithArcadeExpression.QueryFeaturesWithArcadeExpression"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" GeoViewTapped="MyMapView_GeoViewTapped" />
+        <ProgressBar x:Name="ActivityIndicator"
```

Comment:
This is really wide across the middle. It doesnt appear for very long on my system. Maybe we can make it smaller and move it off to a corner?